ecosystem/indexer-grpc/indexer-transaction-generator/README.md

```diff
@@ -4,11 +4,19 @@ This tool is to generate transactions for testing purpose.
 
 ## Usage
 
-`cargo run -- --config example.yaml --output-folder /your_path_to_store_transactions/`
+Under root folder, i.e., `aptos-core`, run
+
+```
+cargo run -p aptos-indexer-transaction-generator -- \
+  --config ecosystem/indexer-grpc/indexer-transaction-generator/template_config.yaml \
+  --output-folder /tmp/ --move-folder-path \
+  ecosystem/indexer-grpc/indexer-transaction-generator/example_scripts
```

Comment:
is move-folder-path still required even though we specific the path for the each script in the config?

Reply:
We talked offline; I shared the reason why we determined the current CLI input. 

TL;DR
To get the root of all move folders we can
1. ask users what the folder name is or
2. enforce the name of the root folder or
3. take a guess that something looks like the root of move folders
4. the current way, very explicit on the input. 


I chose the last one since the user needs to specify something anyway or they need to follow some rules.

Reply:
Fixed now.